javascript/core/intro-js/javascript-basics-part-1.md

```diff
@@ -39,7 +39,7 @@ Like most languages, there are a number of reserved words in JavaScript that you
 
 ### Identifier naming rules
 
-When naming any function, property or variable, the first character must be a letter, $ sign or underscore.
+When naming any function, property or variable, the first character cannot be a letter, the `$` sign or an underscore.
```

Comment:
This isn't true :)

The initial statement was fairly accurate except "letter" might be too generic but that's probably fine for a basics-level insights. (It's actually [any letter from this special group](https://codepoints.net/search?IDS=1))

See: 
- https://stackoverflow.com/questions/1661197/what-characters-are-valid-for-javascript-variable-names
- https://mathiasbynens.be/notes/javascript-identifiers-es6